include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -333,7 +333,7 @@ struct __parallel_scan_submitter<_CustomName, __internal::__optional_kernel_name
             });
         });
 
-        return __future(__final_event, __result_and_scratch);
+        return {std::move(__final_event), std::move(__result_and_scratch)};
```

Comment:
Again, this is all preference, but I much prefer `__future(std::move(__final_event),...` to `{std::move(__final_event),...`, so that if we want to know the type at this location, we don't need to scroll up 100 lines to the function definition.

Reply:
Fixed